server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -21,6 +21,8 @@ EJECT: E J E C T DOT_FS? -> channel(HIDDEN);
 SKIP1 : S K I P '1' DOT_FS? -> skip;
 SKIP2 : S K I P '2' DOT_FS? -> skip;
 SKIP3 : S K I P '3' DOT_FS? -> skip;
+// symbols
+DOUBLEEQUALCHAR : '==';
```

Comment:
What if you move it to the Technical lexer?

Reply:
updated